README.md

```diff
@@ -22,3 +24,19 @@ npm start
 The result will be in `./generated_trusted_nodes.json` file.
 
 If you hit the API limit, you can also add your infura api key to line 5 in the [src/index.ts].
+
+## Airdrop balances
+
+### Methodology
+The final airdrop reward has been calculated as follows: the awarded addresses are all beneficiaries who cashed out cheque from the trusted nodes between the blocks `4283397` and `5130433` including. The total gBZZ amount an address cashed from the trusted nodes has been summed and devided by the total gBZZ amount cashed from trusted nodes by all the addresses. This ratio then indicates how big portion of the 1 000 000 BZZ given awarded address will receive.
```

Comment:
```suggestion
The final airdrop reward has been calculated as follows: the awarded addresses are all beneficiaries who cashed out cheque from the trusted nodes between the blocks `4283397` and `5130433` included. The total gBZZ amount an address cashed out from the trusted nodes has been summed and divided by the total gBZZ amount cashed out from trusted nodes by all the nodes. This ratio then indicates the size of the portion of the 1 000 000 BZZ the given address will receive.
```
I think this sentence needs a big of rework, especially the `The total gBZZ amount an address cashed out from the trusted nodes has been summed and divided by the total gBZZ amount cashed out from trusted nodes by all the nodes.` part.

Reply:
took a little to get it but i think it's ok